symforce/geo/matrix.py

```diff
@@ -480,6 +480,23 @@ def transpose(self) -> Matrix:
         """
         return self.__class__(self.mat.transpose())
 
+    def lower_triangle(self: MatrixT) -> MatrixT:
```

Comment:
Should probably add a unit test for this method, also mention the diagonal is included